include/lk/snappy_boot_v2.h

```diff
@@ -23,6 +23,11 @@
 #define SNAP_BOOTSELECT_VERSION_V2 0x00010010
 #define SNAP_BOOTSELECT_SIGNATURE_RECOVERY ('S' | ('R' << 8) | ('s' << 16) | ('e' << 24))
 
+// devive lock states
+#define DEVICE_STATE_UNKNOW   0  // initial device state at first boot
```

Comment:
UNKNOWN or are the names actually coming from elsewhere?

BTW. why not an enum?

Reply:
This is just helper to define main states. They are not coming from anywhere. Typo was introduced by me :(

Not enum is following implementation syntax of the existing bootloader I use this in.
But equally fair argument would be that values itself should not be even part of the header file and left entirely on bootloader....